interface/usergroup/user_info.php

```diff
@@ -12,32 +12,67 @@
     <link rel="stylesheet" href="<?php echo $css_header;?>" type="text/css">
     <script src="checkpwd_validation.js" type="text/javascript"></script>
     <?php
-      call_required_libraries(['bootstrap', 'jquery-min-1-9-1']);
+      call_required_libraries(array("jquery-min-3-1-1","bootstrap","font-awesome", "iziModalToast"));
       ?>
     <script language='JavaScript'>
       //Validating password and display message if password field is empty - starts
       var webroot='<?php echo $webroot?>';
       function update_password()
       {
-          top.restoreSession();
```

Comment:
Pretty sure this top.restoreSession() is required and it be removed from you side